sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -139,6 +140,10 @@ where
     : WHERE condition=booleanExpression
     ;
 
+returning
+    : RETURNING selectItem (',' selectItem)*
```

Comment:
minor note: maybe instead of going with the `returning` rule, we can better introduce the `selectItems` rule that we can re-use in few places. Then we would avoid this special handling of the rule context, namely `getReturningItems`.  imho, going for the more common `selectItems` denominator gives a bit more beneficial.

otherwise, lgtm

Reply:
Thanks for the feedback. Do you mean something like this: 
'
INSERT INTO table ('(' ident (',' ident)* ')')? insertSource
        onConflict?
        (RETURNING selectItems)?   

returning
    : RETURNING selectItems
    ;
selectItems
    :
    |  selectItem (',' selectItem)*
    ;
'

Reply:
yes, ah but actually it probably won't work as i thought coz of the optional `(RETURNING selectItems)?`, so just ignore the comment

Reply:
👍